api/handlers.go

```diff
@@ -39,16 +40,31 @@ type (
 		Storage           int64  `json:"storageLimit"`
 	}
 
-	// userUpdateData defines the fields of the User record that can be changed
+	// accountRecoveryDTO defines the payload we expect when a user is trying to
+	// change their password.
+	accountRecoveryDTO struct {
+		Token           string `json:"token"`
+		Password        string `json:"password"`
+		ConfirmPassword string `json:"confirmPassword"`
```

Comment:
don't we usually lower case these json prop names?

Reply:
No. We use snake_case in MongoDB field names and camelCase in JSON fields.

Reply:
ok